examples/chip-tool/commands/interactive/InteractiveCommands.cpp

```diff
@@ -114,14 +116,18 @@ bool InteractiveStartCommand::ParseCommand(char * command)
             return true;
         }
 
-        args[argsCount++] = token;
-        token             = strtok(nullptr, " ");
+	char* carg = new char[ arg.size() ];
+	strcpy(carg, arg.c_str() );
+        args[argsCount++] = carg;
     }
 
     ClearLine();
     gIsCommandRunning = true;
     mHandler->RunInteractive(argsCount, args);
     gIsCommandRunning = false;
 
+    while (--argsCount)
```

Comment:
Please add a comment about the fact that this is purposefully not doing the delete for `args[0]`, because otherwise the logic here looks very odd.